lib/Demangling/OldRemangler.cpp

```diff
@@ -1267,6 +1267,18 @@ void Remangler::mangleImplResult(Node *node) {
   mangleChildNodes(node); // impl convention, type
 }
 
+// SWIFT_ENABLE_TENSORFLOW
+void Remangler::mangleImplDifferentiable(Node *node) {
+  // The old mangler does not encode `@differentiable` function types.
+  Buffer << 'd';
+}
+
+// SWIFT_ENABLE_TENSORFLOW
+void Remangler::mangleImplLinear(Node *node) {
+  // The old mangler does not encode `@differentiable(linear)` function types.
```

Comment:
The comment doesn't really match the code.

Reply:
Oops, that's a good point. I'm not sure when this code path (OldRemangler) is even triggered.
I'd like to be able to add a test for this code path (whether the test checks that remangling occurs/doesn't occur) before changing this OldRemangler comment/code.

Reply:
Fixed comments in https://github.com/apple/swift/pull/26600. Linked to [TF-750](https://bugs.swift.org/browse/TF-750).